administrator/components/com_installer/Model/InstallModel.php

```diff
@@ -76,6 +76,9 @@ public function install()
 		// Load installer plugins for assistance if required:
 		PluginHelper::importPlugin('installer');
 
+		// Load overrides plugin.
+		PluginHelper::importPlugin('overrides');
```

Comment:
If we use the plugin group installer, as @laoneo  suggested, we do not need to insert anything here.